blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.57-jm-c",
+  "version": "0.0.57-jm-d",
```

Comment:
is this the published or unpublished one? (I dont' really know how all this works)